quipucords/scanner/network/inspect_callback.py

```diff
@@ -105,6 +106,7 @@ def v2_runner_on_failed(self, result, ignore_errors=False):
 
     def handle_result(self, result):
         """Handle an incoming result object."""
+        input_log.log_ansible_result(result)
```

Comment:
May want to try/catch around this so any IO or other related failure in this flow doesn't kill a scan.

Reply:
Done.